### ISSUE TITLE
Tweak indexes in migrations

### ISSUE THREAD

**Issue by EliuX** [ioet/time-tracker-backend#75]
By default, the containers have a configuration that makes all the items to be indexed. This is great for accessibility but bad for performance. Therefore, it should be good to tweak the configuration of the indexes: we need to index just those fields that are or could be involved in queries required by our biz-logic at the moment or in the next future.
